docs/reference/images/index.md

```diff
@@ -48,3 +48,37 @@ All public images available in {{brand}} support the following image properties:
 * `os_version=${VERSION_ID}`: operating system version, such as `os_version=22.04`
 
 [Other properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html) may also be set on individual images.
+For instance, the `image_original_user` property reflects the username of the default non-root user of the corresponding operating system -- and that piece of information may come in handy in various automation scenarios.
+So, to find out the username of the default non-root user in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image, you can type the following:
+
+```console
+$ openstack image show "Ubuntu 22.04 Jammy Jellyfish x86_64" \
+    -c properties -f json | jq '.[].image_original_user'
+
+"ubuntu"
+```
+
+And what about the same piece of information from the `Debian 11 Bullseye x86_64` image?
+
+```console
+openstack image show "Debian 11 Bullseye x86_64" \
+    -c properties -f json | jq '.[].image_original_user'
+
+"debian"
+```
+
+## Image updates and automation
+
+Images in {{brand}} are updated regularly, and that is something you can deduce from the `replace_frequency` property.
+See, for example, the value of this property in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image:
+
+```console
+$ openstack image show "Ubuntu 22.04 Jammy Jellyfish x86_64" \
+    -c properties -f json | jq '.[].replace_frequency'
+
+"monthly"
```

Comment:
This needs to clarify that per [the SCS reference](https://docs.scs.community/standards/scs-0102-v1-image-metadata/#image-updating), a frequency of `monthly` means that the image is replaced _at least_ once a month, not _no more frequently than_ once a month.

Reply:
Yes, done.